internal/app/ui_driver.go

```
@@ -0,0 +1,881 @@
+		ui:                      ui,
+		terminal:                func() ports.Terminal { return terminal },
+		clock:                   func() ports.Clock { return clk },
+		disableCapabilityProbe:  true,
+		localEnvironment:        launchEnvironmentForConfig(config),
+		remoteEnvironment:       remoteEnvironmentForConfig(config),
+		stateDir:                platform.StateDir,
+	}
+	if options.remote == "" {
+		// A headless named session creates a new local session; an omitted name
+		// retains the ordinary ephemeral attach intent.
+		intent := uint8(protocol.IntentEphemeral)
```

golangci-lint (2.13.2)

[error] unnecessary conversion

(unconvert)